.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,40 @@
-<!-- Please replace #XX below with an existing issue number. Remove the line entirely if none exist. -->
-Fixes #XX
+## Fixes
+<!-- If PR only partly solves an issue, replace 'Fixes' below with 'Partially addresses'. -->
+<!-- If there is no issue being resolved, open one. -->
+Fixes #[issue number] by @[issue author]
 
 ## Description
-<!-- Add your description below. -->
+<!-- Give a clear and concise description of what the pull request does. -->
```

Comment:
This is not, amusingly, very clear or concise to me.  Maybe
```suggestion
<!-- Describe what the pull request does. -->
```
I find instructing someone to be clear and concise rarely helps, they usually think they *are* being clear and concise.

Reply:
I think "concise" makes sense; I don't think everyone aims for being concise by default. I do agree that "clear" seems self-evident.

Reply:
Reworded in 0202b7e.